ModelViewer/enc_temp_folder/93c6327ebe102cff255c5e98134bec4/ViewerGraphicsWindow.cpp

```diff
@@ -0,0 +1,198 @@
+#include "ViewerGraphicsWindow.h"
```

Comment:
Do we want to include this file in the merge request?

Reply:
No, I will remove it